config/jobs/kubernetes/sig-cloud-provider/azure/generate.sh

```diff
@@ -61,6 +62,11 @@ for release in "$@"; do
   if [[ "${release}" == "master" || "${release}" == "1.23" ]]; then
     installCSIdrivers=" ./deploy/install-driver.sh master local,snapshot,enable-avset &&"
   fi
+  if [[ "${release}" == "master" ]]; then
+    installCSIAzureFileDrivers=" ./deploy/install-driver.sh master local &&"
+  else
+    installCSIAzureFileDrivers=" ./deploy/install-driver.sh master local,snapshot,enable-avset &&"
```

Comment:
if it's only testing against k/k branch, I think snapshot is not needed, just use `./deploy/install-driver.sh master local` is ok. and remove `enable-avset`.  `snapshot` is only needed testing against azure file csi driver branch.

Reply:
it looks like this PR just makes changes testing against k/k branch, so just use `./deploy/install-driver.sh master local`, don't need that if - else.

Reply:
done, that looks better, thanks